### PR TITLE
Install pdf kit

### PR DESCRIPTION
**Pull request by TheoBoschetti**

### CODE REVIEW

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: backend/generatePdf.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+var PdfPrinter = require("pdfmake");
+const moment = require("moment");
+const sendMail = require("sendmail")({ silent: true });
+
+function generatePdf(userInfo) {
+  var fonts = {
```
Comment (LoicBrassart) [inline backend/generatePdf.js]: `var` ? Seriously ? I'm hurt ;_;

File: backend/generatePdf.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
+var PdfPrinter = require("pdfmake");
```
Comment (LoicBrassart) [inline backend/generatePdf.js]: Use a `const` or a `let`

**Review by BastienSaulnier — DISMISSED** [review]
SnE

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils

File: backend/generatePdf.js (hunk 1 above)
Comment (TheoBoschetti) [reply backend/generatePdf.js]: Ah zut

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**